docs/how_to/use_estimator.rst

```diff
@@ -0,0 +1,145 @@
+###########################
+Use the Estimator primitive
```

Comment:
The first sentence says about observable. I wonder if the title could be more `Compute expectation value with an Estimator`. I had often been asked how to easily compute an expectation value with qiskit (with opflow it was more work). Having this title leads more towards what they want to do and this shows how. The title otherwise does not offer much - you would already need to know you wanted to use an Estimator unless you were curious what it was for.

Similar comment for Sampler.

Reply:
Yes, for me those work much better as titles.